doc/classes/ItemList.xml

```diff
@@ -81,6 +82,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns the [Color] set by [member set_custom_bg_color]. Default value is [code]Color(0,0,0,0)[/code].
```

Comment:
Copy paste mistake, it should be `set_custom_fg_color`. Same comment for the spaces after `,`.